serverlessworkflow/modules/ROOT/pages/core/working-with-parallelism.adoc

```diff
@@ -139,9 +139,7 @@ curl -X 'POST' \
   'http://localhost:8080/parallel' \
   -H 'accept: */*' \
   -H 'Content-Type: application/json' \
-  -d '{
-  "workflowdata": {}
-}'
+  -d '{}'
```

Comment:
Line 257 here as well.

Reply:
Line 257 is a response. `workflowdata` was removed only from requests.

Reply:
257 is request, see https://github.com/kiegroup/kogito-docs/blame/12782d0927fae1a277a05b9adba77f59606dd716/serverlessworkflow/modules/ROOT/pages/core/working-with-parallelism.adoc#L257

Reply:
Yea, you're right. The line numbers changed and it confused me.
I've pushed the fix. Thank you.